### PR TITLE
Adjust styling for the block navigation.

### LINKED ISSUE #2578: Block navigation goes too narrow when resizing the window small
Steps to reproduce:
1. Resize the window to quite narrow.
2. Observe the results:
Screenshot 2019-06-12 at 16 06 12

Expected:
- The block navigation shouldn't look broken. Not clear what's the best approach here though.

### PR DESCRIPTION
**Pull request by miina**
Fixes #2578.

Adjusts styling for block navigation to avoid it from looking "broken".

### CODE REVIEW

**Comment by miina** [comment]
This PR basically does the 3 following changes:
- Hides the block navigation when the screen width is less than 1000 since it doesn't fit then.
- Moves the block navigation more to the left to give it more room. This will make the title and block navigation not be on the same line for some specific width but perhaps it's better than showing it on top of a Page?
- Shifts the Page more to the right in some specific case to give more room to the block navigation.
- Does all this for when the sidebar is opened only.
- Sets minimum width to block navigation.

This will improve the style and won't show the navigation broken anymore but will cause some shifts for some specific widths. An alternative could be just hiding the block already before (e.g. width 1100) and leaving everything else the same, reverting the changes from this PR.

Thoughts, @swissspidy?

**Comment by swissspidy** [comment]
Looks reasonable at first glance.

I think this fits very well together with #2523. On small sites, instead of just hiding the block navigation, we could display it like the original block navigation with a popover:

Screenshot 2019-06-18 at 17 13 08

On bigger screens, we wouldn't use a popover, but just keep displaying it at the current position.

Of course that can be done in a separate PR specifically for #2523.

**Comment by miina** [comment]
Let's implement #2523 within a separate PR, I could pick that up after the bug fixes.

**Comment by swissspidy** [comment]
Sounds like a plan.

File: assets/src/stories-editor/components/block-navigation/edit.css [hunk 1]
```diff
@@ -4,19 +4,41 @@
...
 }
 
+@media (max-width: 1200px) {
+
+	.is-sidebar-opened #amp-story-block-navigation {
+		left: 0;
+	}
+}
+
+@media (max-width: 1100px) and (min-width: 1001px) {
```
Comment (kienstra) [inline assets/src/stories-editor/components/block-navigation/edit.css]: This isn't wrong, but do you think having the `min-width` before the `max-width` would make it easier to read?

Comment (miina) [reply assets/src/stories-editor/components/block-navigation/edit.css]: Sure, I put it this way thinking of desktop-first, but it's not following this generally. Will change.

**Pull request merged**